src/aws-ec2/instance-connect-endpoint.ts

```diff
@@ -45,6 +45,20 @@ export interface InstanceConnectEndpointProps {
   readonly vpc: aws_ec2.IVpc;
 }
 
+export interface InstanceConnectEndpointAttributes {
```

Comment:
Please add the JSDoc for public factors.

Reply:
I've updated.